personal/wojciech_smolak/reports/cw3_raport.Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "cw3_raport"
+author: "Wojciech Smolak"
+date: "4/25/2020"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(sf)
+library(raster)
+library(spData)
+library(spDataLarge)
+library(dplyr)
+library(ggplot2)
+```
+
+## Zajęcia3::ćwiczenia::1
+
+Zbiór danych world posiada współrzędne geograficzne w stopniach i dziesiętnych częściach stopni. Utwórzcie punkt, który będzie znajdował się gdzieś na terenie Polski i wyświetlcie go na mapie świata.
+
+```{r point creation, echo=FALSE}
+lnd_point = st_point(c(18.002186, 53.131825)) 
+lnd_geom = st_sfc(lnd_point, crs = st_crs(world))          
+lnd_attrib = data.frame(
+  name = "Dom Wojtka",
+  address.street = "Henryka Sienkiewicza",
+  address.number = 30,
+  address.appartment = 4,
+  temperature = 7,
+  date = as.Date("2020-04-25") 
+  )
+lnd_sf = st_sf(lnd_attrib, geometry = lnd_geom) 
```

Comment:
polecam dobrą praktykę korzystania z <- do przypisywania zmiennych, zamiast "=". Sam widzę, że mam to źle w kilku miejscach w prezentacjach, więc poprawię, ale uczulam, że to dobra praktyka rozdzielać = jako przypisywanie argumentu w funkcji i <- jako przypisywanie zmiennej.